Sources/TuistDependencies/Mappers/PruneOrphanExternalTargetsGraphMapper.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import TuistCore
+import TuistGraph
+
+/**
+ External dependencies might contain targets that are only relevant in development, but that
+ that are not necessary when the dependencies are consumed downstream by Tuist projects.
+ This graph mappers detects and prunes those targets
+ */
```

Comment:
This also means we skip them from caching, right? 💯

Reply:
Exactly